nodes/nodes/eco-node-maintenance-operator.adoc

```diff
@@ -50,9 +47,19 @@ include::modules/eco-resuming-node-maintenance-cr-cli.adoc[leveloffset=+2]
 == Gathering data about the Node Maintenance Operator
 To collect debugging information about the Node Maintenance Operator, use the `must-gather` tool. For information about the `must-gather` image for the Node Maintenance Operator, see xref:../../support/gathering-cluster-data.adoc#gathering-data-specific-features_gathering-cluster-data[Gathering data about specific features].
 
+[id="working-with-bare-metal-nodes"]
+== Working with bare-metal nodes
+In addition to maintaining virtual nodes, the Node Maintenance Operator can also maintain bare-metal nodes. For clusters with bare-metal nodes, you can place a node into maintenance mode, and resume a node from maintenance mode by using the web console Actions control.
```

Comment:
For clusters with bare-metal nodes, you can place a node into maintenance mode, and resume a node from maintenance mode by using the web console Actions control. These methods are additional to the above methods and only valid for bare-metal cluster.

Reply:
Then add a NOTE

> These methods, setting and resuming `NodeMaintenance` CR, are additional to the above methods and only valid for bare-metal cluster.

Reply:
@ogradyp @anna-savina WDYT?
I think that without this note the user might think that only the methods under `Working with bare-metal nodes` are valid for bare-metal nodes/cluster.

Reply:
Updated as requested http://file.emea.redhat.com/pogrady/TELCODOCS-28/nodes/nodes/eco-node-maintenance-operator.html#working-with-bare-metal-nodes